Packages/Netherlands3D/BAGInformation/Runtime/BAG/GeoJSONPropertyLoader.cs

```diff
@@ -0,0 +1,114 @@
+*
+*    https://github.com/Amsterdam/Netherlands3D/blob/main/LICENSE.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" basis,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+*  implied. See the License for the specific language governing
+*  permissions and limitations under the License.
+*/
+using Netherlands3D.Events;
+using Netherlands3D.Utilities;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace Netherlands3D.BAGInformation
+{
+	/// <summary>
+	/// Loads GeoJSON from an URL using unique ID's, and invoke events for 
+	/// returned key/value pairs for all properties.
+	/// </summary>
+	public class GeoJSONPropertyLoader : MonoBehaviour
+	{
+		[SerializeField]
+		private string idReplacementString = "{BagID}";
+
+		[Tooltip("Id replacement string will be replaced")]
+		[SerializeField]
+		private string geoJsonRequestURL = "https://service.pdok.nl/lv/bag/wfs/v2_0?SERVICE=WFS&VERSION=2.0.0&outputFormat=geojson&REQUEST=GetFeature&typeName=bag:pand&count=100&outputFormat=xml&srsName=EPSG:28992&filter=%3cFilter%3e%3cPropertyIsEqualTo%3e%3cPropertyName%3eidentificatie%3c/PropertyName%3e%3cLiteral%3e{BagID}%3c/Literal%3e%3c/PropertyIsEqualTo%3e%3c/Filter%3e";
+
+		[Header("Listen to")]
+		[SerializeField]
+		private StringListEvent loadPropertiesForIDs;
+
+		[Header("Invoke")]
+		[SerializeField]
+		private TriggerEvent loadedFeatureWithProperties;
+		[SerializeField]
+		private StringListEvent loadedPropertyKeyValue;
+
+		private Coroutine downloadProcess;
+
+		void Start()
+		{
+			loadPropertiesForIDs.started.AddListener(DownloadGeoJSONProperties);
+		}
+
+		private void OnValidate()
+		{
+			if (!geoJsonRequestURL.Contains(idReplacementString))
+			{
+				Debug.LogWarning("Make sure the url contains the id replacement string", this.gameObject);
+			}
+		}
+
+		private void DownloadGeoJSONProperties(List<string> bagIDs)
+		{
+			if (bagIDs.Count > 0)
+			{
+				var ID = bagIDs[0];
+				if (downloadProcess != null)
+				{
+					StopCoroutine(downloadProcess);
+				}
+				downloadProcess = StartCoroutine(GetIDData(ID));
+			}
+		}
+
+		IEnumerator GetIDData(string bagID)
+		{
+			var requestUrl = geoJsonRequestURL.Replace(idReplacementString, bagID);
+			var webRequest = UnityWebRequest.Get(requestUrl);
+			yield return webRequest.SendWebRequest();
+
+			if (webRequest.result == UnityWebRequest.Result.Success)
+			{
+				GeoJSON customJsonHandler = new GeoJSON(webRequest.downloadHandler.text);
+				while (customJsonHandler.GotoNextFeature())
+				{
+					var properties = customJsonHandler.GetProperties();
+					foreach (KeyValuePair<string, object> propertyKeyAndValue in properties)
+					{
+						FoundProperty(propertyKeyAndValue);
+					}
+
+					if (properties.Count > 0)
+					{
+						loadedFeatureWithProperties.Invoke();
+					}
+				}
+			}
```

Comment:
if not succesfull, maybe trigger an errorEvent? (stringEvent containing reason?)